log.capnp

```diff
@@ -2095,56 +2039,67 @@ struct Event {
     carControl @23 :Car.CarControl;
     plan @24 :Plan;
     liveLocation @25 :LiveLocationData;
-    ethernetData @26 :List(EthernetPacket);
-    navUpdate @27 :NavUpdate;
-    cellInfo @28 :List(CellInfo);
-    wifiScan @29 :List(WifiScan);
-    androidGnss @30 :AndroidGnss;
-    qcomGnss @31 :QcomGnss;
-    lidarPts @32 :LidarPts;
     procLog @33 :ProcLog;
     ubloxGnss @34 :UbloxGnss;
     clocks @35 :Clocks;
     liveMpc @36 :LiveMpcData;
     liveLongitudinalMpc @37 :LiveLongitudinalMpcData;
-    navStatus @38 :NavStatus;
     ubloxRaw @39 :Data;
-    gpsPlannerPoints @40 :GPSPlannerPoints;
-    gpsPlannerPlan @41 :GPSPlannerPlan;
-    applanixRaw @42 :Data;
-    trafficEvents @43 :List(TrafficEvent);
-    liveLocationTiming @44 :LiveLocationData;
-    orbslamCorrectionDEPRECATED @45 :OrbslamCorrection;
-    liveLocationCorrected @46 :LiveLocationData;
     orbObservation @47 :List(OrbObservation);
     gpsLocationExternal @48 :GpsLocationData;
     location @49 :LiveLocationData;
     uiNavigationEvent @50 :UiNavigationEvent;
-    liveLocationKalmanDEPRECATED @51 :LiveLocationData;
-    testJoystick @52 :Joystick;
-    orbOdometry @53 :OrbOdometry;
-    orbFeatures @54 :OrbFeatures;
-    applanixLocation @55 :LiveLocationData;
-    orbKeyFrame @56 :OrbKeyFrame;
     uiLayoutState @57 :UiLayoutState;
-    orbFeaturesSummary @58 :OrbFeaturesSummary;
     driverState @59 :DriverState;
-    boot @60 :Boot;
     liveParameters @61 :LiveParametersData;
-    liveMapData @62 :LiveMapData;
     cameraOdometry @63 :CameraOdometry;
     pathPlan @64 :PathPlan;
-    kalmanOdometry @65 :KalmanOdometry;
     thumbnail @66: Thumbnail;
     carEvents @68: List(Car.CarEvent);
     carParams @69: Car.CarParams;
     frontFrame @70: FrameData; # driver facing camera
     dMonitoringState @71: DMonitoringState;
     liveLocationKalman @72 :LiveLocationKalman;
-    sentinel @73 :Sentinel;
     wideFrame @74: FrameData;
     modelV2 @75 :ModelDataV2;
     frontEncodeIdx @76 :EncodeIndex; # driver facing camera
     wideEncodeIdx @77 :EncodeIndex;
+
+    # debug
+    testJoystick @52 :Joystick;
+
+
+    # legacy + deprecated
+    model @9 :ModelData; # TODO: rename modelV2 and mark this as deprecated
+    liveLocationKalmanDEPRECATED @51 :LiveLocationData;
+    orbslamCorrectionDEPRECATED @45 :Legacy.OrbslamCorrection;
+    liveUIDEPRECATED @14 :Legacy.LiveUI;
+    sensorEventDEPRECATED @4 :SensorEventData;
+    liveEventDEPRECATED @8 :List(Legacy.LiveEventData);
+
+    ethernetData @26 :List(Legacy.EthernetPacket);
+    cellInfo @28 :List(Legacy.CellInfo);
+    wifiScan @29 :List(Legacy.WifiScan);
+
+    # legacy?
+    liveMapData @62 :LiveMapData;
```

Comment:
@HaraldSchafer these all good to deprecate?

Reply:
yes, don't know anything about ethernetData, cellinfo and wifiScan  but others are good to go